cfe_internal/enterprise/federation/federation.cf

```diff
@@ -56,6 +56,8 @@ bundle agent config
         # To ensure we are using a remote hub that's enabled
         if => strcmp( "on", "$(data[remote_hubs][$(remotes)][target_state])" );
       "fingerprints" slist => getvalues( fingerprint );
+      "feeder[$(remotes)]" string => "$(data[remote_hubs][$(remotes)][ui_name])",
```

Comment:
ui_name, isnt this changing to public key sha now?

Reply:
yep. awesome. fixing.